keras/callbacks.py

```diff
@@ -1305,6 +1305,8 @@ class ModelCheckpoint(Callback):
           epochs, the monitored metric may potentially be less reliable (it
           could reflect as little as 1 batch, since the metrics get reset every
           epoch). Defaults to `'epoch'`.
+        start_save: integer that represent the epoch on which we want to start
```

Comment:
This choice of API is ambiguous because `"epoch"` is only one of two possible frequency modes. This argument would not make sense with a different mode.